### PR TITLE
docs: add a few clarifications regarding usage of action

### LINKED ISSUE #49: docs: mention that action does not works for draft releases
I see @vedantmgoyal2009... Please add a dry-run, that wont create the pull request to msft.

Other issues I faced:
 
- See for example:
https://github.com/gerardog/gsudo/actions/runs/3278755892/jobs/5397905542
Error message not very descriptive. Actual problem was that the release was draft. Maybe add a warning to the docs that draft releases are not eligible.

- https://github.com/gerardog/gsudo/actions/runs/3300408456/jobs/5444962383
```
  remote: Permission to gerardog/winget-pkgs.git denied to gerardog.
  fatal: unable to access 'https://github.com/gerardog/winget-pkgs.git/': The requested URL returned error: 403
  HTTP 401: Personal access tokens with fine grained access do not support the GraphQL API (https://api.github.com/graphql)
  Try authenticating with:  gh auth login
```
  Problem with permissions, I will fix them. but action completed as Success, not fail.

_Originally posted by @gerardog in https://github.com/vedantmgoyal2009/winget-releaser/issues/47#issuecomment-1287444015_

### PR DESCRIPTION
**Pull request by gerardog**

### CODE REVIEW

**Comment by vedantmgoyal9** [comment]
@gerardog I've removed the note regarding since the action support other event types as well.

**Comment by gerardog** [comment]
> @gerardog I've removed the note regarding since the action support other event types as well.

Of course it support other event types, where did you get the sense that was implied? I was just stating what may be obvious for you, but not for others, like me, who spent like 4 hours trying to make it work. All your 4 examples use that event, and somehow I didn't realized that was the best way to plug it in. I was biased to add it to my existing pipelines. So the note is putting into words what your example is saying: just create a new workflow. Also, it literally was "A very simple way to...", not saying "the only" one.

**Comment by vedantmgoyal2009** [comment]
@gerardog I've updated again, with some modifications. Can you please review it?

**Comment by vedantmgoyal9** [comment]
@gerardog I've updated again, with some modifications. Can you please review it?

File: README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ WinGet Releaser allows you to automate this process, with pull requests that are
 
 ## Usage ⚒️
 
+- A very simple to **test** the action is to create a **_new_** workflow that triggers on the [`release`][gh-release-event-docs] event with [`released`][gh-release-event-types-docs] event type. It triggers when a release is published, or a pre-release is changed to a release.
```
Comment (gerardog) [inline README.md:31]: simple... way? 
Why test? Wouldn't it work for real releases?

**Comment by gerardog** [comment]
I've decided to close this PR, as I don't understand your edits.
It's your project and you don't need to ask for my review.

**Comment by vedantmgoyal2009** [comment]
@gerardog why did you delete the repository? I was going to push the corrections and merge it.

**Comment by gerardog** [comment]
I usually refrain from editing content, because content is a very subjective matter, and my english is not so good. I did this time because I thought it was kind of a very simple yet impactful edit.

I added two paragraphs with two concepts lacking in the docs. 
   1- new workflow is easier than modifying your preexisting's workflows. 
   2- Doesn't work with drafts.

You deleted one of them, no reason given, and changed the second one in a way that made no sense to me. So I added a comment, but you didn't answered it. I considered that was rude and not helpful.

This was my first and second edit
image

image

This was your version.
image

Creating a new workflow is not just for testing, can be a final implementation. Why did you changed "A very simple way to use Winget Releaser" to "A very simple to test Winget Releaser" ??? All your edit were, in my opinion, deteriorating my addition. 

It was obvious we were not having fluid communication and my changes were indirectly rejected so I just closed my PR and retired from the conversation. You can do any edit on your repo, no need to ask me.

**Comment by vedantmgoyal9** [comment]
Changing "A very simple way to use Winget Releaser" to "A very simple to test Winget Releaser" was not my intention. I wanted to write "A very simple **way** to test Winget Releaser", but somehow "way" got missed while modifying the sentence.

> Creating a new workflow is not just for testing, can be a final implementation.

From your review comment (`Why test? Wouldn't it work for real releases?`), I realized that my modifications were actually not making any sense. You also closed your PR, but since it was late at night in my area, I just re-opened the PR so that I can look into it. Today, as soon as I started committing changes to re-add that statement, you closed the PR. When I re-opened it, you closed it again by deleting the forked repository.

Surely, it was my fault that I could have replied in the "review comment" section, but due to less time and backlog from other projects, I decided to correct the changes and merge them directly. I did not reject the changes at all. If they were rejected, I wouldn't re-open the PR every time you closed it. At last, I will be more than happy if you would open a new PR adding the changes under "Getting Started" section.

**Comment by gerardog** [comment]
Sure, sorry for all the drama. Let's start again.

**Pull request closed without merging**